app/src/layouts/kanban/kanban.vue

```diff
@@ -223,7 +224,7 @@ const reorderGroupsDisabled = computed(() => !props.canReorderGroups || props.se
 										<div class="bottom">
 											<display-datetime
 												v-if="element.date"
-												format="short"
+												:format="fieldDisplay.dateField?.options?.format ?? 'short'"
```

Comment:
Instead of applying only the format option, let’s apply all display options when the display is set to `datetime`.

```suggestion
												v-bind="
													fieldDisplay.dateField?.display === 'datetime'
														? fieldDisplay.dateField.options
														: { format: 'short' }
												"
```